hoprd/rest-api/src/prometheus.rs

```diff
@@ -18,6 +18,9 @@ lazy_static::lazy_static! {
         &["endpoint", "method"]
     )
     .unwrap();
+
+    // Matches Ed25519-based peer IDs and channel IDs (Keccak256 hashes)
+    static ref ID_REGEX: regex::Regex = regex::Regex::new(r"(0x[0-9A-Fa-f]{64})|(12D3KooW[A-z0-9]{44})").unwrap();
```

Comment:
What if we change a PeerId format? There should be a test for this regex to match against a PeerId produced by our setup.

Reply:
If it fails to match, then nothing will be replaced and we will notice in the metrics.

But out of all the PeerID formats, we're using the only sensible one.

I rather see our shift from PeerIDs completely once we need to become PQ secure.